source/nvwave.py

```diff
@@ -76,17 +75,7 @@ class WAVEFORMATEX(Structure):
 	]
 
 
-LPWAVEFORMATEX = POINTER(WAVEFORMATEX)
-
-
 WAVE_FORMAT_PCM = 1
-WAVE_MAPPER = -1
-
-CALLBACK_NULL = 0
-# CALLBACK_FUNCTION = 0x30000
-CALLBACK_EVENT = 0x50000
-# waveOutProc = CFUNCTYPE(HANDLE, UINT, DWORD, DWORD, DWORD)
-# WOM_DONE = 0x3bd
```

Comment:
as these are API breaking changes, can you please add a corresponding changelog item?